config/config.go

```diff
@@ -0,0 +1,176 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/pflag"
+	c "nav/constants"
+)
+
+type Config struct {
+	ConfValues ConfValues
+	// add DB connection instance here
```

Comment:
Why, it's part of ConfValues already, isn't it? Or is that something else?

Reply:
Actually, I'm not sure I understand the benefit of this indirection. Why not just use the ConfValues struct but call it Config?

Reply:
Currently the DB connection is happening in main():

```
	t := connectToken{conf.ConfValues.DBDriver, conf.ConfValues.DBDSN}
	db := connectDb(&t)

```
My thinking was that it would make more sense if the db connection instance lived together with the configuration logic, and was just passed around the rest of the program as a pointer. So the plan was to move the connection logic to the config package as part of another PR and have the instance here.

Reply:
Ah ok, so that is a TODO for later and "DB connection instance" != "DB instance", as I had thought.

Reply:
I have created an issue for this, closing this conversation as resolved

Reply:
https://github.com/AC-FuSa-Tools/nav/issues/25